src/elements/content-sidebar/__tests__/SidebarPanels.test.js

```diff
@@ -144,4 +149,21 @@ describe('elements/content-sidebar/SidebarPanels', () => {
             expect(instance.versionsSidebar.current.refresh).toHaveBeenCalledWith();
         });
     });
+
+    test('should reset the current version if the versions route is no longer active', () => {
+        const onVersionChange = jest.fn();
+        const wrapper = getWrapper({ location: { pathname: '/activity' }, onVersionChange });
+
+        wrapper.setProps({ location: { pathname: '/activity/versions/123' }, isOpen: true });
```

Comment:
Can we add assertions for `/activity/versions` and `/details/versions`, as well?